<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>661</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Video</string>
    </property>
    <widget class="QWidget" name="ImgWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>64</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="videoSource">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>64</width>
      <height>32</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>No Source</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sourceComboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maxVisibleItems">
     <number>5</number>
    </property>
    <item>
     <property name="text">
      <string>Desktop Video Source</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IPCam Video Source</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>File Attachment Source</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No Source</string>
     </property>
    </item>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Labels</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Objects</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Measure</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Vectors</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtering</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Matching</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cropping</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>16</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="sourceComboBox_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maxVisibleItems">
     <number>5</number>
    </property>
    <item>
     <property name="text">
      <string>Desktop Video Source</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IPCam Video Source</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>File Attachment Source</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No Source</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="delay">
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>0</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
